.gitignore

```diff
@@ -15,5 +15,9 @@
 *.idea
 *.vscode
 
+# Working files for Postman
+/tools/postman/openapi/*
```

Comment:
Question: is there any benefit to ignore results of the folder if we intend to keep the atlas-api file there?

Might be worth to think of 2 separate folders:

- `openapi` folder that is not ignored
- `bin` folder containing the "compiled" results

No strong opinion from my side - only thinking of desired folder structure.

Reply:
I was considering having a tmp folder for the temporary files. I agree having the two folders for different uses is clearer

Reply:
Really like idea of `tmp` or `bin` folder. It does send the message. 
[Golang folder structure for inspiration](https://github.com/golang-standards/project-layout).